spec/factories/case_information.rb

```diff
@@ -28,6 +24,15 @@
 
     association :team, code: '1234', name: 'A nice team'
 
-    crn { Faker::Alphanumeric.alpha(number: 10) }
+    crn do Faker::Alphanumeric.alpha(number: 10) end
+
+    probation_service do
+      'Wales'
+    end
+
+    trait :no_team do
+      probation_service { 'Scotland' }
```

Comment:
Is 'Scotland' an allowed value for the `probation_service` field?

The CaseInformation model has a validation rule to only allow 'England' and 'Wales'.

So unless I'm missing something, I think the `:no_team` trait will create a CaseInformation record which could never exist in the wild.

Reply:
Actually – scratch that. I see that the additional allowed values ('Scotland', 'Northern Ireland') were added to the validation rule in the next commit.